src/bin/unit_test_map.c

```diff
@@ -165,7 +167,7 @@ int main(int argc, char *argv[])
 	}
 #endif
 
```

Comment:
If we're going to be producing an 'ok' file, you need to unlink it at the start of the utility's execution.  If the file doesn't exist, the resulting error should be ignored.

Removing the file at the start, means if the utility is manually executed and then fails, make test will run the test again.